build.sbt

```diff
@@ -0,0 +1,46 @@
+import play.sbt.routes.RoutesKeys._
+import sbt.Keys._
+import sbt._
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.DefaultBuildSettings.defaultSettings
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
+
+val appName = "ras-frontend"
+
+lazy val plugins: Seq[Plugins] =
+  Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+
+lazy val playSettings: Seq[Setting[_]] = Seq.empty
+
+lazy val scoverageSettings = {
+  Seq(
+    ScoverageKeys.coverageExcludedPackages := "<empty>;testOnlyDoNotUseInAppConf.*;conf.*;models.*;views.*;app.*;uk.gov.hmrc.*;prod.*",
+    ScoverageKeys.coverageMinimum := 70,
+    ScoverageKeys.coverageFailOnMinimum := false,
+    ScoverageKeys.coverageHighlighting := true,
+    parallelExecution in Test := false
+  )
+}
+
+lazy val microservice = Project(appName, file("."))
+  .enablePlugins(plugins: _*)
+  .settings(
+    majorVersion := 0,
+    playSettings,
+    scoverageSettings,
+    publishingSettings,
+    defaultSettings(),
+    scalaVersion := "2.11.11",
```

Comment:
remove this, use the scalaSettings from default build settings: https://github.com/hmrc/pbik-stub/blob/e380a23b99f5ee4633eb41df421765ba4af1353c/build.sbt#L14

This will deal with the scala version for us

Reply:
done